chat-android/src/main/java/com/ably/chat/Messages.kt

```diff
@@ -239,7 +239,13 @@ internal class DefaultMessages(
      */
     private val messagesChannelName = "$roomId::\$chat::\$chatMessages"
 
-    override val channel: Channel = realtimeChannels.get(messagesChannelName, ChatChannelOptions())
+    override val channel = realtimeChannels.get(messagesChannelName, ChatChannelOptions())
+
+    override val contributor: ContributesToRoomLifecycle = this
+
+    override val attachmentErrorCode: ErrorCodes = ErrorCodes.MessagesAttachmentFailed
```

Comment:
As per https://github.com/ably-labs/ably-chat-kotlin/pull/33/files#r1834803690, errorCode value is assigned here with proper type checking.